src/components/PreviewComment/index.js

```diff
@@ -108,7 +108,7 @@ export const styles = StyleSheet.create({
   },
   container: (totalComment) => ({
     borderLeftWidth: 1,
-    marginHorizontal: SIZES.base - 1,
+    marginHorizontal: SIZES.base - 2,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in `marginHorizontal` from `SIZES.base - 1` to `SIZES.base - 2` might affect the layout of components using this style. Please ensure that this change doesn't unintentionally affect other components' layouts.
```diff
-    marginHorizontal: SIZES.base - 1,
+    marginHorizontal: SIZES.base - 2,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->